src/server/common/test-helpers/journey-state.js

```diff
@@ -101,7 +101,7 @@ export const applicationStateWithAnimalIdentifiersSection = {
   origin: {
```

Comment:
This variable probably wants renaming from `applicationStateWithAnimalIdentifiersSection`, (and 'identification' removed), since that section is no longer part of the application state. 

Should have caught this in a previous PR ! but only noticed it now.